__tests__/app.test.js

```diff
@@ -37,24 +38,37 @@ describe("GET /api/topics", () => {
 });
 
 describe("GET /api/articles", () => {
-  test("200: Responds with an array of all article objects", () => {
+  test("200: Responds with an array of all article objects, incl comments_count, sorted by date", () => {
     return request(app)
     .get("/api/articles")
     .expect(200)
-    .then(({ body: { articles } }) => {
+    .then(({ body: {articles} }) => {
+
         expect(articles.length).toBeGreaterThan(0);
 
+        expect(articles).toBeSortedBy('created_at', {
```

Comment:
considering this is a different behaviour, it could be separated into its own test 👍